barman/backup_executor.py

```diff
@@ -671,6 +674,28 @@ def backup_copy(self, backup_info):
             else:
                 _logger.debug(msg)
 
+    def _raise_summarize_wal_value_from_parent_backup(self, parent_backup_info):
+        """
+        Status can be "on" | "off" | None
+
+        :param barman.infofile.LocalBackupInfo parent_backup_info: parent backup information
+        :raises tuple[BackupException,PostgresUnsupportedFeature]: if a command argument is considered invalid
+        """
+        if parent_backup_info.summarize_wal:
+            if parent_backup_info.summarize_wal == "off":
+                raise BackupException(
+                    "summarize_wal is 'off' for the parent backup id. "
+                    "Executing incremental backups require for it to be "
+                    "set to 'on' in your PostgreSQL server before taking "
+                    "the first full backup of the chain."
+                )
+        else:
+            raise PostgresUnsupportedFeature(
```

Comment:
Does it make sense to check both None and "off"? And do the errors make sense as well? @gcalacoci @barthisrael @gustabowill @bleidens

Reply:
Here i thought about splitting because None is a PG_VERSION issue and off would be just a config issue!

Reply:
Maybe we can simply test `!= "on"` through this PR.
There is another card to verify the version, which will likely be one more `if` statement around the logic provided by this PR.

Reply:
not a fan of `PostgresUnsupportedFeature` in this context. I would use  `BackupException` here too... this is a backup "lifecycle" error... not a postgres one. if you get what I mean.

Reply:
Yes, thats what it thought @barthisrael ! The pg_version would be higher in the validation! and yes, i got what you mean @gcalacoci! I will leave just one then!